sandbox/src/Bitcoin.tsx

```diff
@@ -232,6 +232,87 @@ function BtcSignPSBT({ bb02 }: Props) {
   );
 }
 
+function BtcSignMessage({ bb02 }: Props) {
+  const [simpleType, setSimpleType] = useState<bitbox.BtcSimpleType>('p2wpkhP2sh');
+  const [keypath, setKeypath] = useState("m/49'/0'/0'/0/0");
+  const [message, setMessage] = useState('message');
+  const [running, setRunning] = useState(false);
+  const [result, setResult] = useState<bitbox.BtcSignMessageSignature | undefined>();
+  const [err, setErr] = useState<bitbox.Error>();
+
+  const coin = 'btc';
+  const simpleTypeOptions: bitbox.BtcSimpleType[] = ['p2wpkhP2sh', 'p2wpkh'];
+
+  const scriptConfig: bitbox.BtcScriptConfig = { simpleType };
+  const scriptConfigWKeypath: bitbox.BtcScriptConfigWithKeypath = { scriptConfig: scriptConfig, keypath: keypath };
```

Comment:
`scriptConfig: scriptConfig` => `scriptConfig` now after the rename :grin:

Reply:
True, using the shorter notation in both of the keys then.